todo_app/trello_board_actions.py

```diff
@@ -71,12 +73,12 @@ def delete_card(self, card_id):
     def update_card_status(self, id):
         "Update card status"
         new_list_id = 0
-        if self.is_card_on_list("to-do", id):
-            new_list_id = self.get_list_id_by_name("doing")
-        elif self.is_card_on_list("doing", id):
-            new_list_id = self.get_list_id_by_name("done")
+        if self.is_card_on_list("To Do", id):
```

Comment:
It would be nice to put these strings ("To Do", "Doing", "Done") into constants or an Enum

Reply:
Can include these comments in my Module 4 PR